colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -6,7 +6,8 @@
       "name": "W&B-RayTune.ipynb",
```

Comment:
<p>Drop the <code># Quickstart</code> -- that's just for our Simple Integrations.</p><p> </p><p>You want a space after the <code>#</code> in front of the title, so it looks like:</p><h1>Weights &amp; Biases 💜 Ray/Tune</h1><h1><br></h1><p>Love the text!</p><p> </p><p>I have a few suggested changes (additions in <strong>bold</strong>; subtractions in <s>strikethrough</s>). Watch to make sure you're consistent with <strong>Ray/Tune</strong> rather than <strong>Ray Tune</strong> (with a space).</p><p> </p><blockquote><s>Whereas </s>W&amp;B <strong>(add link)</strong> is a <s>centralized repository for</s> <strong>toolkit with </strong>everything you need to track, reproduce and gain insights from your models easily; Ray/Tune <strong>(add link)</strong> provides a simple interface for scaling and running distributed experiments. 🤝<strong>They're a natural match!🤝</strong></blockquote><blockquote>(<strong>added a newline here</strong>)</blockquote><blockquote><strong>Here's just a</strong> few reasons why our community <strong>💜s </strong>Ray/Tune –</blockquote><blockquote>Simple <s>D</s><strong><s>d</s></strong>istributed execution: Ray<strong>/</strong>Tune makes it easy to scale <strong>all the way </strong>from a single <s>node</s> <strong>laptop</strong><s>, to multiple GPUs, and further multiple nodes.</s> <strong>multiple nodes in a cluster each with multiple GPUs</strong></blockquote><blockquote>Large number of algorithms: Ray<strong>/</strong>Tune has a huge number of algorithms including Population Based Training (<strong>add link)</strong>, ASHA <strong>(add link)</strong>, and HyperBand <strong>(add link)</strong></blockquote><blockquote><s>Framework</s>Method agnostic: Ray<strong>/</strong>Tune works across <strong>deep learning</strong> frameworks <strong>(</strong>including PyTorch, Keras, Tensorflow, <s>XGBoost, and </s>PyTorchLightning<strong>) and with other ML methods like gradient-boosted trees (XGBoost, LightGBM).</strong></blockquote><blockquote>Fault-tolerance: Ray<strong>/</strong>Tune is built on top of Ray, providing fault tolerance out of the box.</blockquote><blockquote><br></blockquote><blockquote><strong>This Colab demonstrates how this integration works for a simple grid search over two hyperparameters. If you've got any questions about the details,</strong></blockquote><blockquote><strong>check out </strong><a href="https://docs.wandb.com/library/integrations/ray-tune" target="_blank"><strong>our documentation</strong></a><strong> or the </strong><a href="https://docs.ray.io/en/master/tune/api_docs/integration.html#weights-and-biases-tune-integration-wandb" target="_blank"><strong>documentation for Ray/Tune</strong></a><strong>.</strong></blockquote><blockquote>Let's start by in<s>a</s>stalling the libraries.</blockquote><p> </p><p>I would suggest bolding the parts in the list before the colons.</p><p> </p><p>Some of this text sounds familiar. Is this copy-pasted from something by Ray? If so, we'll need to paraphrase, as I did with the first line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>